base/common/python/pki/nssdb.py

```diff
@@ -1378,7 +1375,7 @@ def export_cert_from_db(self,
         if output_format:
             cmd.extend(['--format', output_format])
 
-        cmd.extend([full_name, output_file])
+        cmd.extend([nickname, output_file])
```

Comment:
When you say that the `serverCertNick.conf` already has the prefix added, which file are you referring to? I don't see any processing of nickname for hsm environment

Reply:
During installation we store the cert nickname and the token name ([if available](https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/instance.py#L519-L520)) into these files:
* [`/etc/pki/<instance>/serverCertNick.conf`](https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/instance.py#L522), and
* [`/etc/pki/<instance>/server.xml`](https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/instance.py#L532)

During startup the nickname and the token name (if available) will be [reloaded](https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/cli/migrate.py#L141) from either of those files using [`PKIInstance.get_sslserver_cert_nickname()`](https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/instance.py#L497) as a single value, so it's not necessary to prepend the value with the token name again.

Note that eventually we want to remove `serverCertNick.conf`, but that hasn't been implemented yet.

Reply:
@edewata Ah, good catch. Thanks!